architecture/networking/routes.adoc

```diff
@@ -296,8 +296,10 @@ Path based routes specify a path component that can be compared against
 a URL (which requires that the traffic for the route be HTTP based) such
 that multiple routes can be served using the same host name, each with a
 different path. Routers should match routes based on the most specific
-path to the least; however, this depends on the router implementation. The
-following table shows example routes and their accessibility:
+path to the least; however, this depends on the router implementation. 
+The host name and path are passed through to the server so it should be
+prepared to succesfully answer requests for them. 
```

Comment:
"prepared to succesfully answer requests for them." 
wording seems a little clumsy to me. spelling "succesfully" -> successfully